# Enforce a case style for dirnames (@sserov/dirnames/match-kebab-case)

Enforces all linted dirnames to have their names in kebab case style
## Cases

### `kebabCase`

- `src/foo-bar`
- `src/my-component`

## Rule Details

Examples of **incorrect** dirnames for this rule:

```
/src/fooBar/index.js
```

Examples of **correct** dirnames for this rule:

```
/src/foo-bar/index.js
```
